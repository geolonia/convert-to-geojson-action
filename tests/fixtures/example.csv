title,description,lat,lng,marker-size,marker-color,stroke,stroke-width
シンデレラ城,シンデレラ城（Cinderella Castle）は、東京ディズニーランドのシンボルであるランドマーク。最高地点の高さは51メートルで、頭頂部には園内業務用無線のアンテナが設置されている。 ,35.63209422,139.8808906,large,#FF0000,#FFFFFF,2