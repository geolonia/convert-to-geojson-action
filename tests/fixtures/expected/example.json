{"type":"FeatureCollection","features":[{"type":"Feature","properties":{"title":"シンデレラ城","description":"シンデレラ城（Cinderella Castle）は、東京ディズニーランドのシンボルであるランドマーク。最高地点の高さは51メートルで、頭頂部には園内業務用無線のアンテナが設置されている。 ","marker-size":"large","marker-color":"#FF0000","stroke":"#FFFFFF","stroke-width":"2"},"geometry":{"type":"Point","coordinates":[139.8808906,35.63209422]}}]}